searchcore/src/vespa/searchcore/proton/server/bootstrapconfig.cpp

```diff
@@ -56,7 +61,9 @@ BootstrapConfig::valid() const
     return _documenttypes.get() != NULL &&
                     _repo.get() != NULL &&
                   _proton.get() != NULL &&
-      _tuneFileDocumentDB.get() != NULL;
+        _fileDistributorRpc.get() != NULL &&
+        _bucketspaces.get() != NULL &&
+        _tuneFileDocumentDB.get() != NULL;
```

Comment:
These can be upgraded to `nullptr`

Reply:
I guess even better to just use the boolean operators and just remove the whole ".get() != NULL"

Reply:
This is addressed in PR #4599